plugin/gcp/storageObjectPrefix.go

```diff
@@ -0,0 +1,42 @@
+package gcp
+
+import (
+	"context"
+
+	"cloud.google.com/go/storage"
+	"github.com/puppetlabs/wash/plugin"
+)
+
+type storageObjectPrefix struct {
+	plugin.EntryBase
+	bucket *storage.BucketHandle
+	prefix string
+}
+
+// Takes the name of the directory, as well as the full prefix path.
+// Attrs may be nil if they could not be retrieved. Some prefixes don't appear to have attributes.
+func newStorageObjectPrefix(bucket *storage.BucketHandle,
```

Comment:
Strange that prefixes here have attributes but S3 prefixes don't.

Reply:
Yeah. Surprised me.